app/grandchallenge/participants/models.py

```diff
@@ -54,3 +56,88 @@ def delete(self, *args, **kwargs):
 
     class Meta:
         unique_together = (("challenge", "user"),)
+
+
+def string_type_schema():
+    return {"type": "string"}
+
+
+class RegistrationQuestion(UUIDModel):
+    challenge = models.ForeignKey(
+        Challenge,
+        on_delete=models.CASCADE,
+    )
+
+    question_text = models.TextField(blank=False)
+    question_help_text = models.TextField(blank=True)
+
+    schema = models.JSONField(
+        default=string_type_schema,
```

Comment:
Does a default of string make sense? Will we have to coerse answers later on?

Reply:
Requirements state that the validation should default to a string by default. I thought the cleanest way was to add a string schema.

I am unsure what you mean with answer cooercion later on. The actual HTML inputs (and hence POST) will contain only strings and hence will need to be coerced into pythonic objects before fed to this database field.